domains/_vercel.alora.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "aloramiaa",
+    "email": "xaloramia@gmail.com",
+    "discord": "1236292707371057216"
+  },
+  "record": {
+    "TXT": "vc-domain-verify=drive.alora.is-a.dev,395400702612ae6db6a8"
```

Comment:
TXT records should be in an array as opposed to two objects which is invalid JSON.

Reply:
It works even in a string though?

Reply:
Nvm just realised its 2 strings